internal/operators/lvm/common_test.go

```diff
@@ -0,0 +1,95 @@
+package lvm
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var hwRequirement = `
+[
+	{
+	  "version": "default",
+	  "master": {
+		"cpu_cores": 1,
+		"ram_mib": 2,
+		"disk_size_gb": 3,
+		"installation_disk_speed_threshold_ms": 10,
+		"network_latency_threshold_ms": 100,
+		"packet_loss_percentage": 0
+	  },
+	  "worker": {
+		"cpu_cores": 2,
+		"ram_mib": 2,
+		"disk_size_gb": 3,
+		"installation_disk_speed_threshold_ms": 10,
+		"network_latency_threshold_ms": 1000,
+		"packet_loss_percentage": 10
+	  },
+	  "sno": {
+		"cpu_cores": 3,
+		"ram_mib": 2,
+		"disk_size_gb": 3,
+		"installation_disk_speed_threshold_ms": 10,
+		"network_latency_threshold_ms": 1000,
+		"packet_loss_percentage": 10
+	  }
+	}
+]`
+
+func mock_get_page(url string) (*http.Response, error) {
```

Comment:
```suggestion
func mockGetPage(url string) (*http.Response, error) {
```

let's be consistent for the naming of the functions